config/materials/ultracur3d_rg35.mtl

```diff
@@ -0,0 +1,92 @@
+﻿ND_RelParSet_K01 = {
+
+Name = ULTRACUR3D_RG35
+
+PARAMETERS =
+{
+  Name = PTC_MATERIAL_TYPE
+  Type = Integer
+  Default = 9
+  Access = Locked
+},
+{
+  Name = PTC_FAILURE_CRITERION_TYPE
+  Type = String
+  Default = 'NONE'
+  Access = Locked
+},
+{
+  Name = PTC_FATIGUE_TYPE
+  Type = String
+  Default = 'NONE'
+  Access = Locked
+},
+{
+  Name = PTC_MATERIAL_SUB_TYPE
+  Type = String
+  Default = 'LINEAR'
+  Access = Locked
+},
+{
+  Name = PTC_MATERIAL_DESCRIPTION
+  Type = String
+  Default = 'ABS-like for SLA
+Source: datasheet from Xometry'
+  Access = Full
+},
+{
+  Name = TEMPERATURE
+  Type = Real
+  Default = 0.000000e+00 C
+  Access = Full
+},
+{
+  Name = PTC_INITIAL_BEND_Y_FACTOR
+  Type = Real
+  Default = 5.000000e-01
+  Access = Full
+},
+{
+  Name = PTC_BEND_TABLE
+  Type = String
+  Default = ''
+  Access = Full
+},
+{
+  Name = PTC_MASS_DENSITY
+  Type = Real
+  Default = 1.200000e-06 kg / mm^3
+  Access = Full
+},
+{
+  Name = PTC_YOUNG_MODULUS
+  Type = Real
+  Default = 2.600000e+06 kg / (mm sec^2)
+  Access = Full
+},
+{
+  Name = PTC_TENSILE_ULTIMATE_STRESS
+  Type = Real
+  Default = 6.200000e+01 MPa
```

Comment:
I read this value instead:

![immagine](https://github.com/user-attachments/assets/a19784f0-6f28-453e-ab3c-4ec5c992264e)